README.md

```diff
@@ -95,6 +95,8 @@ You can join the team by posting a comment to
 | [![nabdelgadir]](https://github.com/nabdelgadir) | [![marioestradarosa]](https://github.com/marioestradarosa) |        [![yaty]](https://github.com/yaty)        |
 |                 Dominique Emond                  |                         Agnes Lin                          |                 Deepak Rajamohan                 |
 |     [![emonddr]](https://github.com/emonddr)     |         [![agnes512]](https://github.com/agnes512)         | [![deepakrkris]](https://github.com/deepakrkris) |
+|                  Denny Bartelt                   |                      Doug "dougal83"                       |                  Rifa Achrinza                   |
```

Comment:
Please update name: `"Doug "dougal83"` to `Douglas McConnachie`

Reply:
@dougal83 It's been updated